tools/pd-simulator/simulator/cases/balance_leader.go

```diff
@@ -58,10 +61,18 @@ func newBalanceLeader(config *sc.SimConfig) *Case {
 		})
 	}
 
-	simCase.Checker = func(regions *core.RegionsInfo, _ []info.StoreStats) bool {
-		for i := 1; i <= totalStore; i++ {
-			leaderCount := regions.GetStoreLeaderCount(uint64(i))
-			if !isUniform(leaderCount, totalRegion/totalStore) {
+	simCase.Checker = func(stores []*metapb.Store, regions *core.RegionsInfo, _ []info.StoreStats) bool {
```

Comment:
Do we still needs this checker?

Reply:
yes, I think we can keep the logic for previous cases